src/client/components/lazy-chat/providers/crisp.js

```diff
@@ -18,6 +20,14 @@ export function onAccepted () {
  */
 export function onLoaded () {}
 
+function onOpened () {
```

Comment:
I'd rather keep the `ga` implementation out of the `crisp` provider. So can do to the same here as you do `onSessionLoaded` in the chat icon feature? Expose `onOpened` and `onClosed` as methods on this provider and make them accept a callback. In `lazy-chat` you can then use these methods and do the tracking.